pages/about.vue

```diff
@@ -170,24 +185,23 @@
       </v-row>
 
       <!-- colabs row-->
-      <v-row class="pa-12" id="collaborator-projects">
+      <v-row :class="pageSectionClass" id="collaborator-projects">
         <v-col class="mb-10" :md="3" :sm="8">
           <h3 class="pt-6">Collaborator Partners</h3>
         </v-col>
 
         <div class="d-md-flex justify-md-space-between align-center">
           <v-col
-            :md="2"
-            :sm="6"
+            cols="6"
             v-for="(collab, i) in collaborator_projects"
             :key="i"
             @mouseover="collab.src = collab.color"
             @mouseleave="collab.src = collab.grey"
           >
             <a :href="collab.url">
               <img class="collab-img" :src="collab.src" />
-              <div class="caption d-flex justify-start collab-name">
-                <div style="font-size:16px;">{{ collab.name }}</div>
+              <div class="caption d-flex justify-start">
+                <div style="font-size: 16px;">{{ collab.name }}</div>
```

Comment:
Rather than setting this here, use one of the typography classes? `body-2` might work?